tests/test_pourbaix_diagram.py

```diff
@@ -0,0 +1,356 @@
+from __future__ import annotations
+import pymatgen
+import pyEQL
+import multiprocessing
+from unittest import TestCase
+
+import matplotlib.pyplot as plt
+import numpy as np
+from monty.serialization import dumpfn, loadfn
+from pytest import approx
+
+from pyEQL.Pourbaix.pourbaix_diagram import (
+    IonEntry,
+    MultiEntry,
+    PourbaixDiagram,
+    PourbaixEntry,
+    PourbaixPlotter,
+    ion_or_solid_comp_object,
+)
+from pymatgen.core.composition import Composition
+from pymatgen.core.ion import Ion
+from pymatgen.entries.computed_entries import ComputedEntry
+from pymatgen.util.testing import TEST_FILES_DIR, PymatgenTest
+
+TEST_DIR = f"{TEST_FILES_DIR}/analysis/pourbaix_diagram"
+
```

Comment:
It looks like this test calls some additional files from the pymatgen TEST_FILES_DIR, so those files will need to be copied into our tests directory, and this path updated